tests/testthat/test-with-abort.txt

```diff
@@ -46,8 +45,8 @@ fields:  `message`, `trace`, `parent` and `error`
 backtrace:
     █
  1. ├─rlang::with_abort(f())
- 2. │ └─base::withCallingHandlers(...)
- 3. └─rlang:::f()
- 4.   └─rlang:::g()
- 5.     └─rlang:::h()
- 6.       └─base::stop("Low-level message")
+ 2. │ ├─rlang::eval_bare(rlang::expr(withCallingHandlers(expr, !!!handlers)))
```

Comment:
Maybe it's worth generating the expression in a separate line just to make this slightly cleaner?

Reply:
Now also `.Call()`s into `Rf_eval()` to make that line disappear completely.